sdk/Lusid.Sdk.Tests/tutorials/Instruments/FutureExamples.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Lusid.Sdk.Model;
+using Lusid.Sdk.Tests.Utilities;
+using Lusid.Sdk.Utilities;
+using LusidFeatures;
+using NUnit.Framework;
+
+namespace Lusid.Sdk.Tests.Tutorials.Instruments
+{
+    [TestFixture]
+    public class FutureExamples: DemoInstrumentBase
+    {
+        internal override void CreateAndUpsertMarketDataToLusid(string scope, ModelSelection.ModelEnum model, LusidInstrument instrument)
+        {
```

Comment:
Consider adding comment for why this is empty

Reply:
Same as the other one. I think we would have to populate market data. More just refactoring for now.